src/pages/about.js

```diff
@@ -24,10 +24,24 @@ module.exports = React.createClass({
       aboutCopy: aboutCopy
     });
   },
+  imageTest: function() {
+    if (this.props.test === "img-a") {
+      return (<picture>
+        <source srcSet="/assets/images/homepage-images/teacher-wide.jpg" media="(max-width: 800px)" />
+        <img className="homepage-image" src="/assets/images/homepage-images/teacher-full.jpg" alt="Man pointing at student's computer screen"/>
```

Comment:
I notice that this image no longer prespecifies its size, which the original still does. If we can, can we prespecify the dimensions again?

Reply:
It's not clear the best way to do this with responsive images. Punting for now, as this is a bit of a micro-optimization